RevenueCatUI/Templates/V2/Components/Button/BottomSheetView.swift

```diff
@@ -100,6 +101,7 @@ struct BottomSheetOverlayModifier: ViewModifier {
                         viewModel: sheetViewModel.sheetStackViewModel,
                         onDismiss: {
                             self.sheetViewModel = nil
+                            onDismiss?()
```

Comment:
Hmm actually... if someone adds a close button to the sheet content, we probably would only want to dismiss the sheet I guess? This seems like a behavior breaking change as well....

Reply:
Great catch Toni! you are probably right!
It's confusing though, in the sense that close buttons in paywalls are supposed to close the paywall BUT I totally get that people would want to add the close button to close the sheet only 🤔

Reply:
Yeah... I'm not sure of the best way to fix this... We could differentiate between "close entire paywall" and "close this screen" events I guess, but seems a bit more work involved...

Reply:
Yep. I'd also argue that this all should be transparent in the dashboard. I.e. we should implement this in the SDK: so that a close button inside a bottom sheet only closes the sheet (as it is currently) but the auto dismiss dismisses the whole paywall (the current auto dismiss logic in all places other than in the sheet).